inputfiles/knownTypes.json

```diff
@@ -0,0 +1,92 @@
+{
+    "Window": [
```

Comment:
i actually meant the new `window` types.. i understand the the worker, but not sure i understand the new window types.

Reply:
It's mainly because we cannot automatically track `override-signatures` and `override-type`s. Additionally there are some types that are not directly referenced in IDL by other types but still used by specs, e.g. subdictionaries of RTCStats.

Reply:
Can not we compute this by walking the types after we merge in addedTypes?

Reply:
We already do after addedTypes but the two `override` types are not in IDL structures but in TypeScript string format. Maybe we still can extract type names there, but I'm not sure it'll worth the effort as they ultimately will be removed...

Reply:
fair enough. can you resolve the merge conflicts.